milestone-3/frontend/src/components/recipePreview.tsx

```diff
@@ -10,7 +10,7 @@ function RecipePreview(props: Recipe) {
     return (
     <div className="sub-section">
     <h2>
-        <Link to={`/recipes/${props.name}`} className="text">{props.name}</Link>
+        <Link to={`/recipePage/${props.name}`} className="text">{props.name}</Link>
     </h2>
     <div className="sub-section-content">
         <img className="recipe-images-home" src={props.image} alt="picture of banana bread" width="200" height="200"/>
```

Comment:
this alt text is not accurate for all the other recipes that aren't banana bread. Instead you should put something dynamic as the alt text that is unique to each one. (maybe reuse the name or description from the props you pass down.